health/health.go

```diff
@@ -213,8 +213,10 @@ type Warnable struct {
 	// Deprecated: this is only used in one case, and will be removed in a future PR
 	MapDebugFlag string
 
-	// If true, this warnable is related to configuration of networking stack
-	// on the machine that impacts connectivity.
+	// If true, this Warnable is related to configuration of networking stack
```

Comment:
field comment should start with the name of the field.

    // ImpactsConnectivity is whether ...

Reply:
done

Reply:
can we get a linter check for this at some point?